tests/pyfunc/test_model_export_with_loader_module_and_data_path.py

```diff
@@ -786,6 +788,107 @@ def test_log_model_persists_requirements_in_mlflow_model_directory(
     assert pyfunc_custom_env_dict["dependencies"][-1]["pip"] == requirements
 
 
+@pytest.mark.large
+def test_check_requirements_and_local_installed_mismatch(sklearn_knn_model):
+    import cloudpickle
+    import mlflow.utils.requirements_utils
+
+    with mlflow.start_run() as run:
+        mlflow.sklearn.log_model(sklearn_knn_model, "model")
+        mlflow.sklearn.log_model(
+            sklearn_knn_model,
+            "model2",
+            pip_requirements=[
+                "scikit-learn>=0.8,<=0.9",
+                "nonexist-pkg==1.2.3",
+            ],
+        )
+        pyfunc_model_path = _download_artifact_from_uri(
+            "runs:/{run_id}/{artifact_path}".format(run_id=run.info.run_id, artifact_path="model")
+        )
+        pyfunc_model2_path = _download_artifact_from_uri(
+            "runs:/{run_id}/{artifact_path}".format(run_id=run.info.run_id, artifact_path="model2")
+        )
```

Comment:
Do we need to log a model? Can we just create a requirements file that looks like this:

```
scikit-learn>=0.8,<=0.9
nonexist-pkg==1.2.3
```